index.html

```diff
@@ -382,12 +384,12 @@
         <pre class="example">
 &lt;!-- controller.html --&gt;
 &lt;head&gt;
-  &lt;!-- the link element with rel='default-presentation' allows the page to specify --&gt;
-  &lt;!-- the presentation URL and id for when the UA initiates a presentation session --&gt;
-  &lt;link href="http://example.com/presentation.html" rel="default-presentation" &gt;
+  &lt;!-- Setting presentation.defaultRequest allows the page to specify the
+       PresentationRequest to use when the UA initiates a presentation session --&gt;
 &lt;/head&gt;
 &lt;script&gt;
-  navigator.presentation.ondefaultsessionstart = function (evt) {
+  navigator.presentation.defaultRequest = new PresentationRequest(defaultUrl);
+  navigator.presentation.defaultRequest.onsessionconnect = function(evt) {
```

Comment:
We use the `sessionavailable` event on NavigatorPresentation to indicate when a session has connected to the presented page [1].  It would seem more economical to use the same event to indicate that the controlling page has connected to the presented page and vice versa.  At first glance I prefer the name `sessionconnect` for both; I will file a followup issue.

[1] https://w3c.github.io/presentation-api/#monitoring-incoming-presentation-sessions-in-a-presenting-browsing-context